x-pack/solutions/security/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -2279,6 +2279,17 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'windows.advanced.events.security.event_disabled',
+    first_supported_version: '9.2.0',
```

Comment:
if i understood correctly, we're adding this to 8.19.6, skipping 9.0 and 9.1, and afterwards it is available from 9.2.0. then i assume it is safe to say that the first supported version is `9.2.0`, but maybe it'd be worth to add some explanation to the description about this extra support in 8.19.6 to help the users understand which Endpoint versions they can use this on. what do you think?